libraries/ESP8266WiFi/src/WiFiUdp.cpp

```diff
@@ -84,7 +84,18 @@ uint8_t WiFiUDP::begin(uint16_t port)
     return (_ctx->listen(IPAddress(), port)) ? 1 : 0;
 }
 
+uint8_t WiFiUDP::beginMulticast(IPAddress multicast, uint16_t port)
+{
+    IPAddress local = WiFi.localIP();
```

Comment:
`IP_ADDR_ANY`?

Reply:
I'm not an expert of multicast UDP and I don't have an opinion about that, so that's sounds good to me.
I have done my search for other usages in the framework, and found both `IP_ADDR_ANY` and `localIP`. BTW since the [doc](https://arduino-esp8266.readthedocs.io/en/latest/esp8266wifi/udp-class.html) is currently suggesting to use `localIP()`, I believe that this was the correct way. Changing that.